sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionTestBase.scala

```diff
@@ -26,6 +26,7 @@ import scala.collection.mutable
 import scala.util.Random
 
 import org.apache.spark.sql.catalyst.{QualifiedTableName, TableIdentifier}
+import org.apache.spark.sql.catalyst.catalog.CatalogTypes.TablePartitionSpec
```

Comment:
Could you fix the import order to pass the GitHub Action and Jenkins?

Reply:
forgot to fix this one. thanks for reminding!